### DISCUSSION TITLE
Graveyard maps labelled incorrectly

### DISCUSSION THREAD

**Discussion by osulover9000** [ppy/osu#18201] [Q&A] [answered]
### Type

Other

### Bug description

Many graveyard maps are instead labelled as not submitted, despite them being submitted on the website.

### Screenshots or videos

https://user-images.githubusercontent.com/104596572/167484621-ee7c46a8-35ab-4fa7-906a-0e0d00c9fb70.mp4

### Version

Latest AppImage

### Logs

N/A

**Comment by smoogipoo** [comment]
I'm not sure what I'm looking at, is the issue that there's no "graveyard" pill on the beatmap panel and wedge? We don't yet have a process for updating the status of beatmaps after import.

**Comment by peppy** [comment]
Are these potentially beatmaps that you imported into lazer a long time ago? They may not have got correct metadata on import. Could you try re-downloading the beatmaps in question and see if it resolves the issue when they import a second time?

**Comment by osulover9000** [comment]
I imported all of my maps less than a month ago. Deleting maps and importing them by dragging and dropping only works if I'm signed in, and there are too many maps for me to re-download. It doesn't seem to make sense that there are still some graveyard maps despite me never being signed in while importing maps, which is where this issue seems to stem from.

**Reply by peppy** [reply]
That sounds wrong. The import process should still do beatmap lookups when not signed in. Will check on that.

**Comment by peppy** [comment, marked as answer]
Okay so the case is definitely when you are not logged in. We are tracking this via https://github.com/ppy/osu/issues/4823. Eventually, the game will re-fetch correct information after you login, but currently this is not a thing.

For now, please login before importing beatmaps.
